Acceleration docs (#114)
spiceai/docs: spiceaidocs/content/en/reference/Spicepod/datasets.md

## `acceleration.enabled`

Enable or disable acceleration, defaults to `true`.

## `acceleration.engine`

The acceleration engine to use, defaults to `arrow`. The following engines are supported:

>   - `arrow` - Cache data as in-memory Apache Arrow Records.

Reviewer: Remove the word Cache.

Use "Accelerated in-memory backed by Apache Arrow DataTables`

>   - `duckdb` - Cache data in a DuckDB.

Reviewer: "Accelerated by an embedded DuckDB database" (with link to DuckDB)

>   - `postgres` - Cache data in a PostgreSQL database.

Reviewer: "Accelerated by a PostgreSQL database"

## `acceleration.mode`

Optional. The mode of acceleration. The following values are supported:

>   - `memory` - Cache data in memory.

Reviewer: ```suggestion
  - `memory` - Store acceleration data in-memory.
```

>   - `file` - Cache data in a file.

Reviewer: ```suggestion
  - `file` - Store acceleration data in a file.
```

> `mode` is only supported for the `duckdb` engine.

Reviewer: ```suggestion
`mode` is currently only supported for the `duckdb` engine.
```

Optional. Only supported for `append` datasets. Specifies how long to retain data updates from the data source before they are deleted.

If not specified, the default retention is to keep all data.

## `acceleration.params`

Optional. Parameters to pass to the acceleration engine. The parameters are specific to the acceleration engine used.

## `acceleration.engine_secret`

> Optional. The secret to use to connect to the acceleration engine. Use `spice login` to create a new secret.

Reviewer: ```suggestion
Optional. The secret store key to use the acceleration engine connection credential. For supported data connectors, use `spice login` to store the secret.
```
